src/lib.rs

```diff
@@ -72,9 +72,11 @@ use alloc::{
 pub use ttf_parser::Language;
 pub use ttf_parser::Width as Stretch;
 
+use slotmap::DenseSlotMap;
```

Comment:
Why `DenseSlotMap` specifically? For faster iteration? I think just a `SlotMap` should be fine. Unless you have some benchmarks.

Reply:
No reason; it's just a good default. I've switched it to normal `SlotMap`; I'm sure we can benchmark it later.